jinja2/base.html

```diff
@@ -70,16 +84,14 @@
     {% set favicon_suffix = '-local' %}
   {% endif %}
   <link rel="shortcut icon" href="{{ static('img/favicon32' + favicon_suffix + '.png') }}">
-  <!--[if IE]>
-  <meta http-equiv="imagetoolbar" content="no">
-  {% javascript 'html5shiv' %}
-  <![endif]-->
 
   {% javascript 'font-check' %}
 
   {% block extrahead %}{% endblock %}
 
-  {% include "includes/google_analytics.html" %}
+  <![if !lte IE 9]>
+    {% include "includes/google_analytics.html" %}
```

Comment:
I'm 👎 on removing GA from IE 9. It seems it would be useful to continue to get a browser count.

Reply:
Yup, that makes sense. I was thinking that as well. I am working on some tweaks on this PR, and will also create a small JS bundle to send some(the current) metrics across.